pootle/apps/pootle_store/forms.py

```diff
@@ -517,6 +519,27 @@ def clean_month(self):
     def clean_user(self):
         return self.cleaned_data["user"] or self.request_user
 
+    def clean_path(self):
+        if self.cleaned_data["path"].strip("/") in ["", "projects"]:
```

Comment:
`split_pootle_path()` already provides the stripping being done here — as I see only the two first items it returns are needed in this context.
```python
language_code, project_code = split_pootle_path(self.cleaned_data['path'])[:2]
```